mysql/tables.sql

```diff
@@ -99,4 +99,36 @@ CREATE TABLE user_tags (
   KEY tag_id (tag_id),
   CONSTRAINT user_tags_ibfk_1 FOREIGN KEY (user_id) REFERENCES users (id),
   CONSTRAINT user_tags_ibfk_2 FOREIGN KEY (tag_id) REFERENCES tags (id)
-) ENGINE=InnoDB DEFAULT CHARSET=utf8;
\ No newline at end of file
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+
+-- Alter database for new tables
+ALTER DATABASE
+    bread_server
```

Comment:
@Areezy Each user will name their local db something unique, can you replace this to use the environment variable for db? `process.env.LOCAL_DB`

Reply:
That has been done now. Thanks for the heads-up!